src/test/java/StringCalculatorTest.java

```diff
@@ -0,0 +1,75 @@
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class StringCalculatorTest {
+
+// 요구 사항 정의
+
+
+
+    @Test
+    @DisplayName("null 또는 빈문자열 입력시 0 반환")
+    void null_and_empty(){
+        // 구분자 구하기
+        assertAll(
+                () -> assertThat(StringCalculator.cal(null)).isEqualTo(0)
+                , () -> assertThat(StringCalculator.cal("")).isEqualTo(0)
+        );
+    }
+
+    @Test
+    @DisplayName(", 구분자로 입력할 경우")
+    void comma_delimeter() throws Exception {
+        // given
+        String target = "1,2";
+        // when
+        int result = StringCalculator.cal(target);
+        // then
+        Assertions.assertThat(result).isEqualTo(3);
+    }
+    @Test
+    @DisplayName("구분자 여러개가 입력된 경우")
+    void comma_and_dot_delimeter() {
+        // given
+        String target = "1,2:3";
+        // when
+        int result = StringCalculator.cal(target);
+        // then
+        Assertions.assertThat(result).isEqualTo(6);
+    }
+    @Test
+    @DisplayName("구분자 여러개가 입력된 경우")
+    void with_custom_delimeter() {
+        // given
+        String target = "//:\n1:2:3";
+        // when
+        int result = StringCalculator.cal(target);
+        // then
+        Assertions.assertThat(result).isEqualTo(6);
+    }
+
+    @Test
+    @DisplayName("마이너스가 들어갔을 경우")
+    void minus_number_delimeter() {
+        // given
+        String target = "-1,2,3";
+        // then
+        org.junit.jupiter.api.Assertions.assertThrows(RuntimeException.class,
+                () -> StringCalculator.cal(target));
+    }
+    @Test
+    @DisplayName("숫자 하나를 입력했을 경우")
+    void just_one_number() {
+        // given
+        String target = "1";
+        // then
+        int result = StringCalculator.cal(target);
+        Assertions.assertThat(result).isEqualTo(1);
```

Comment:
```suggestion
        // given
        String target = "1";
        
        // when
        int result = StringCalculator.cal(target);
        
        // then
        Assertions.assertThat(result).isEqualTo(1);
```

검증하고자 하는 메소드가 무엇인지 표출하면 좋을것 같아요!

Reply:
이 말이 이전 피드백에 남겨주셨던 given when then 구조를 공백으로 확실하게 구분해서 구조를 확실히 하자와 비슷한 의미일까요?